study-builder/studies/rarex/general-information.conf

```diff
@@ -2098,20 +2098,199 @@
                       ]
                     }
                   ]
-                },
-                "detailLabelTemplate": {
+                }
+              },
+              {
+                "stableId": "PREFER_NOT_TO_ANSWER",
+                "optionLabelTemplate": {
                   "templateType": "TEXT",
-                  "templateText": "$none_of_above_details",
+                  "templateText": "$general_information_gender_prefer_not_to_answer",
                   "variables": [
                     {
-                      "name": "none_of_above_details",
+                      "name": "general_information_gender_prefer_not_to_answer",
                       "translations": [
-                        { "language": "en", "text": "" }
+                        { "language": "en", "text": ${i18n.en.general_information_gender_prefer_not_to_answer} }
                       ]
                     }
                   ]
-                },
-                "allowDetails": true
+                }
+              }
+            ]
+          },
+          "blockType": "QUESTION",
+          "shownExpr": """!user.studies["rarex"].forms["ADD_PARTICIPANT"].questions["PARTICIPANT_RELATIONSHIP"].answers.hasOption("PARENT")"""
+        },
+        {
+          "question": {
+            include required("../snippets/text-question.conf"),
+            "hideNumber": true,
+            "promptTemplate": {
+              "templateType": "TEXT",
+              "templateText": "",
+              "variables": []
+            },
+            "placeholderTemplate": {
+              "templateType": "TEXT",
+              "templateText": "$general_information_gender_other_detail",
+              "variables": [
+                {
+                  "name": "general_information_gender_other_detail",
+                  "translations": [
+                    {
+                      "language": "en",
+                      "text": ${i18n.en.general_information_gender_other_detail}
+                  }
+                  ]
+                }
+              ]
+            },
+            "validations": [
+              {
+                "ruleType": "REQUIRED",
+                "hintTemplate": {
+                  "templateType": "TEXT",
+                  "templateText": "$general_information_gender_other_detail_req_hint",
+                  "variables": [
+                    {
+                      "name": "general_information_gender_other_detail_req_hint",
+                      "translations": [
+                        {
+                          "language": "en",
+                          "text": ${i18n.en.general_information_gender_other_detail_req_hint}
+                      }
+                      ]
+                    }
+                  ]
+                }
+              },
+              {
+                "ruleType": "LENGTH",
+                "maxLength": 50,
```

Comment:
Let's make this 500. See Slack [here](https://broadinstitute.slack.com/archives/C01A6HWK3PX/p1620833065217400?thread_ts=1620832797.216900&cid=C01A6HWK3PX). Can you also update the other limits, such as for the "other detail" for "race" question? Thanks!

Reply:
Done